cmd/basecli/commands/cmds.go

```diff
@@ -103,39 +111,53 @@ func readSendTxFlags(tx *btypes.SendTx) error {
 
 /******** AppTx *********/
 
+// BroadcastAppTx wraps, signs, and executes an app tx basecoin transaction
+func BroadcastAppTx(tx *btypes.AppTx) (*ctypes.ResultBroadcastTxCommit, error) {
```

Comment:
Yes, good helper.  Creating the tx from input is app-dependent, as is the output logic, but this should be the same for almost every app.